src/app/games/reversi/tests/ReversiHeuristic.spec.ts

```diff
@@ -21,88 +17,52 @@ describe('ReversiHeuristic', () => {
-            expect(boardValue).toBe(16);
-        });
-        it('should get 4 points for edges', () => {
-            const board: Table<PlayerOrNone> = [
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, X, O, _, _, _],
-                [_, _, _, O, X, _, _, _],
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, X],
-                [_, _, _, _, _, _, _, _],
-            ];
-            const state: ReversiState = new ReversiState(board, 1);
-            const node: ReversiNode = new ReversiNode(state);
-            const boardValue: number = heuristic.getBoardValue(node).value;
-            expect(boardValue).toBe(4);
-        });
-        it('should get 1 points for normal square', () => {
-            const board: Table<PlayerOrNone> = [
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, X, O, _, _, _],
-                [_, _, _, O, X, _, _, _],
-                [_, _, _, _, _, _, _, _],
-                [_, _, _, _, _, _, X, _],
-                [_, _, _, _, _, _, _, _],
-            ];
-            const state: ReversiState = new ReversiState(board, 1);
-            const node: ReversiNode = new ReversiNode(state);
-            const boardValue: number = heuristic.getBoardValue(node).value;
-            expect(boardValue).toBe(1);
-        });
-    });
-});
-
-
-describe('ReversiMinimax', () => {
-
-    let rules: ReversiRules;
-    let minimax: Minimax<ReversiMove, ReversiState, ReversiLegalityInformation>;
-    const minimaxOptions: AIDepthLimitOptions = { name: 'Level 2', maxDepth: 2 };
-
-    beforeEach(() => {
-        rules = ReversiRules.get();
-        minimax = new Minimax('Minimax', ReversiRules.get(), new ReversiHeuristic(), new ReversiMoveGenerator());
-    });
-    it('should not throw at first choice', () => {
-        const node: ReversiNode = rules.getInitialNode();
-        const bestMove: ReversiMove = minimax.chooseNextMove(node, minimaxOptions);
-        expect(rules.isLegal(bestMove, ReversiState.getInitialState()).isSuccess()).toBeTrue();
+    it('should get 16 points for corner', () => {
+        const board: Table<PlayerOrNone> = [
+            [_, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _],
+            [_, _, _, X, O, _, _, _],
+            [_, _, _, O, X, _, _, _],
+            [_, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, _],
+            [_, _, _, _, _, _, _, X],
+        ];
+        const state: ReversiState = new ReversiState(board, 1);
+        const node: ReversiNode = new ReversiNode(state);
+        const boardValue: number = heuristic.getBoardValue(node).value;
+        expect(boardValue).toBe(16);
     });
-    it('should prioritize taking control of the corners', () => {
```

Comment:
tu l'as mise dans Minimax ? C'est bien du behavior :p !

Reply:
Reformulé et bougé dans les tests d'heuristique !

Reply:
à renommer sur place en "should prefer owning the corners" but qu'il s'agit pas de choix de move